src/coreclr/tools/aot/ILCompiler.RyuJit/Compiler/RyuJitCompilationBuilder.cs

```diff
@@ -66,12 +67,18 @@ public override CompilationBuilder UseILProvider(ILProvider ilProvider)
             return this;
         }
 
+        public CompilationBuilder UseResilience(bool resilient)
```

Comment:
My suggestion was to move UseResilience from coreclr\tools\aot\ILCompiler.ReadyToRun\Compiler\ReadyToRunCodegenCompilationBuilder.cs to this file to CompilationBuilder.cs. That way we only have one UseResilience method and field that is shared with crossgen2 (that file is shared with cg2).